### PR TITLE
Using larger instances for their memory and specifying AZ

### PR DESCRIPTION
**Pull request by cberkhoff**

### CODE REVIEW

**Review by akoshelev — APPROVED** [review]
Can you make a comment why and when we should go back to c6id.8xlarge?

File: eksctl/cluster-config.yaml [hunk 1]
```diff
@@ -8,21 +8,22 @@ metadata:
...
     labels: { releaseName: h3 }
-    instanceType: c6id.8xlarge
+    instanceType: c6id.12xlarge
     desiredCapacity: 1
   - name: rc
     labels: { releaseName: rc }
     instanceType: c6id.8xlarge
     volumeSize: 2000
     desiredCapacity: 1
+availabilityZones: ['us-east-1a']
```
Comment (akoshelev) [inline eksctl/cluster-config.yaml:28]: why they are all in the same AZ?

Comment (cberkhoff) [reply eksctl/cluster-config.yaml:28]: 2 things: 
1. its better for networking speed
2. 1a is the larger region, with this constrain we avoid getting hosts in one of the smaller regions.

**Comment by cberkhoff** [comment]
> Can you make a comment why and when we should go back to c6id.8xlarge?

sure

**Pull request merged**